sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidgetListener.kt

```diff
@@ -0,0 +1,17 @@
+package com.mapbox.maps.renderer.widget
+
+import android.graphics.Bitmap
+import com.mapbox.maps.MapboxExperimental
+
+/**
+ * Listener for BitmapWidget events.
+ */
+@MapboxExperimental
+fun interface BitmapWidgetListener {
+  /**
+   * Invoked when the supplied Bitmap has been loaded.
+   * The Bitmap object is no longer required by the BitmapWidget,
+   * and can be recycled if needed during the callback.
+   */
+  fun onLoaded(bitmap: Bitmap)
```

Comment:
What is the reason for exposing this? It looks like good functionality to recycle the bitmap when it has loaded onto the map.

But why expose this to downstream? It seems a developer should "give ownership" of the bitmap to the `BitmapWidget`